stdlib/public/stubs/CMakeLists.txt

```diff
@@ -18,6 +18,18 @@ set(LLVM_OPTIONAL_SOURCES
     ${swift_stubs_objc_sources}
     ${swift_stubs_unicode_normalization_sources})
 
+# ICU isn't required on Darwin, but is on every other platform.
+# Now in case we're cross-compiling from Darwin for another platform,
+# the find_package should still be executed.
+if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
+  set(icu_required, "REQUIRED")
+else()
+  set(icu_required, "")
+endif()
+
+find_package(ICU ${icu_required} COMPONENTS uc)
+set(ICU_UC_LIBRARY "")
```

Comment:
This seems at odds with what Android and the newer port-to-Windows work (cc @modocache again and @hughbe). I don't understand what the 'REQUIRED' is doing, either—the code doesn't match the comment, which doesn't seem to be the desired behavior anyway.

(Also, CMake doesn't use commas.)